api/spec/requests/ransackable_attributes_spec.rb

```diff
@@ -69,7 +69,7 @@
 
       get "/api/products?q[name_cont]=fritos", token: user.spree_api_key
 
-      product_names = JSON.parse(response.body)['products'].map { |product| product['name'] }
+      product_names = JSON.parse(response.body)['products'].map { |x| x['name'] }
```

Comment:
I prefer `x`, because `p` is http://ruby-doc.org/core-2.3.0/Kernel.html#method-i-p